.github/workflows/testNginxForAzureDeploy.yml

```
@@ -3,63 +3,87 @@
 name: Test Github action to update NGINX for Azure configurations
 on:
   schedule:
-    - cron:  '0 20 * * *'
+    - cron: "0 20 * * *"
 
 env:
   NGINX_DEPLOYMENT_NAME: github-action-test-dep
   NGINX_TRANSFORMED_CONFIG_DIR_PATH: /etc/nginx/
   NGINX_ROOT_CONFIG_FILE: nginx.conf
   TEST_RESOURCE_GROUP_NAME: testenv-0da38993-workload
+  NGINX_CERT_NAME: github-action-test-crt
+  NGINX_VAULT_NAME: nlbtest-customer
 
 permissions:
   id-token: write
   contents: read
 
 jobs:
-  Update-NGINX-Configuration:
+  Update-NGINX:
     runs-on: ubuntu-latest
     steps:
-      - name: 'Checkout repository'
+      - name: "Checkout repository"
         uses: actions/checkout@v2
-      - name: 'AZ CLI Login'
+      - name: "AZ CLI Login"
         uses: azure/login@v1
         with:
-            client-id: ${{ secrets.AZURE_CLIENT_ID }}
-            tenant-id: ${{ secrets.AZURE_TENANT_ID }}
-            subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
-      - name: 'Update config - single file'
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      - name: "Sync NGINX Certificate to NGINX on Azure instance - single cert"
```

Nit,

```suggestion
      - name: "Sync NGINX Certificate to NGINX for Azure - single cert"
```

Updated product name to match marketplace and removed the word "instance" as it's meaning may be ambiguous.